### PR TITLE
[Merged by Bors] - [cu-363aqgb] Bootstrapping script package to create testing data.

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
*Please replace this line with either a link to the ClickUp issue, or a reference to the GitHub issue.*

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Review by andor0 — APPROVED** [review]

File: scripts/bootstrap-pallets/interfaces/assets/definitions.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      type: "Vec<Asset>"
+    }
+  },
+  types: {
+    Asset: {
+      name: "Vec<u8>",
+      id: "u64"
+    }
+  }
+};
```
Comment (KaiserKarel) [inline scripts/bootstrap-pallets/interfaces/assets/definitions.ts:40]: Can we obtain this from composable.js @Dom-Roth

File: scripts/bootstrap-pallets/interfaces/assets/index.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// Auto-generated via `yarn polkadot-types-from-defs`, do not edit
+/* eslint-disable */
+
+export * from "./types";
```
Comment (KaiserKarel) [inline scripts/bootstrap-pallets/interfaces/assets/index.ts:4]: Same, can we use composable.js?

File: scripts/bootstrap-pallets/src/bootstrap/assets/index.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { mintAssetsToAddress } from "@bootstrap-pallets/lib";
+import { ApiPromise } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+export async function bootstrapAssets(api: ApiPromise, walletSudo: KeyringPair, mint: [string, string, string][]): Promise<void> {
+    for (const m of mint) {
+        const [wallet, assetId, amount] = m;
```
Comment (KaiserKarel) [inline scripts/bootstrap-pallets/src/bootstrap/assets/index.ts:7]: amount is a `string`?

File: scripts/bootstrap-pallets/src/bootstrap/assets/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { mintAssetsToAddress } from "@bootstrap-pallets/lib";
+import { ApiPromise } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+
+export async function bootstrapAssets(api: ApiPromise, walletSudo: KeyringPair, mint: [string, string, string][]): Promise<void> {
```
Comment (KaiserKarel) [inline scripts/bootstrap-pallets/src/bootstrap/assets/index.ts:5]: nit: would love a struct for the mint function

File: scripts/bootstrap-pallets/src/bootstrap/assets/index.ts (hunk 3 above)
Comment (saadjhk) [reply scripts/bootstrap-pallets/src/bootstrap/assets/index.ts:7]: yes as it comes from the JSON

File: scripts/bootstrap-pallets/src/bootstrap/assets/index.ts (hunk 4 above)
Comment (saadjhk) [reply scripts/bootstrap-pallets/src/bootstrap/assets/index.ts:5]: @KaiserKarel you mean object with named keys?

File: scripts/bootstrap-pallets/interfaces/assets/index.ts (hunk 2 above)
Comment (saadjhk) [reply scripts/bootstrap-pallets/interfaces/assets/index.ts:4]: yeah i would be moving this to composable.js folder once that PR is merged

**Pull request closed without merging**